testing/simulator/src/eth1_sim.rs

```diff
@@ -18,8 +18,12 @@ use std::time::Duration;
 use tokio::time::sleep;
 use types::{Epoch, EthSpec, MinimalEthSpec};
 
-const FORK_EPOCH: u64 = 2;
 const END_EPOCH: u64 = 16;
+const ALTAIR_FORK_EPOCH: u64 = 0;
+const BELLATRIX_FORK_EPOCH: u64 = 1;
```

Comment:
It kind of seems like these should be dynamic rather than constants, even when `--post-merge-sim` is off we'll still fork to Bellatrix, and by forking Altair at epoch 0 we lost the ability to test the Altair transition

Reply:
I'm not sure how having them dynamic would be useful though :thinking:  Having it as `ALTAIR_FORK_EPOCH=1` and `BELLATRIX_FORK_EPOCH=2` seems good enough to me to test both the transitions.

Reply:
won't this be inadvertently testing our behaviour with offline EEs in the non-merge tests though? like we'll fork to Bellatrix but not have any online EEs

Reply:
Ohh yaa I forgot it'll still make requests to the EE without the ttd set. I'll fix it, thanks :)